### PR TITLE
Placeholder for empty description added. (fixes #305)

### LINKED ISSUE #305: Description text box doesn't appear on detail page when "no description" for workitem
Description text box doesn't appear on detail page when "no description" for workitem
When there is no description for a workitem , description box no appear on screen (when you click on details view page).
Expected : Description text box should be visible on detail page
Actual: Its hidden When there is no description and appear if you click on that area
screenshot from 2016-11-11 13-51-37
screenshot from 2016-11-11 13-54-11

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by sanbornsen** [comment]
@nimishamukherjee @naina-verma @mindreeper2420 @joshuawilson Please review. I am glad that Naina raised an issue regarding this.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html [hunk 1]
```diff
@@ -98,15 +98,32 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
...
                 (keyup.enter)="preventDef($event)"
                 id="wi-detail-desc" 
                 class="description"
-                [innerHTML]="workItem.fields['system.description']">
+                [innerHTML]="
+                  workItem.fields['system.description'] ? 
+                    workItem.fields['system.description'] : 
+                    descEditable ? 
+                      workItem.fields['system.description'] : 
+                      'No description found for this work item.'">
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html]: Is this the text that displays if there is no description entered? If not, it should be 'Work item Description', as in the mockups.

Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html]: Yeah, this is the text shows up as place holder. I'll change it to 'Work item Description'.

**Comment by nimishamukherjee** [comment]
What is the expected behaviour when
- we have a work item with no description
- the text "No description found for this work item" displayed
- I click edit icon to edit the description
- I don't enter any text and click on Save
Current behaviour - Nothing happens on-screen. Shouldn't save work as description is not compulsory field.

nodescclick

**Comment by sanbornsen** [comment]
It goes back to showing you the place holder.

**Comment by mindreeper2420** [comment]
@nimishamukherjee - @sanbornsen is correct, in that it should revert back to the default placeholder state.

**Comment by nimishamukherjee** [comment]
In that case, I'm fine with this PR @sanbornsen @mindreeper2420 .

**Pull request merged**